modules/exploits/linux/http/trueonline_p660hn_v2_rce.rb

```diff
@@ -98,8 +98,8 @@ def exploit
     @cookie = rand_text_alpha_lower(7)
 
     res = send_request_cgi({
+      'method' => "GET",
```

Comment:
It means to add `vars_get` and `vars_post`, not `GET` and `POST` methods, which would be impossible as far as sanity goes.